build/xtask/src/dist.rs

```diff
@@ -2512,10 +2500,17 @@ fn write_elf(
             p_paddr: base as u64,
             p_filesz: sec.data.len() as u64,
             p_memsz: sec.data.len() as u64,
-            p_align: 0x1000,
+            p_align: 0x20, // This matches the alignment guarantees of the kernel & task build
```

Comment:
Is this coming from the max-page-size and common-page-size in `link` and `build`?

Reply:
Yes, I believe so. For future work, we could explore changing all these alignments to be a target-specific property (likely as part of a wider refactor of xtask - the signing flow in particular is extremely device-specific)